### DISCUSSION TITLE
pg_explainer.py

### DISCUSSION THREAD

**Discussion by zuluokonkwo** [pyg-team/pytorch_geometric#8072] [Q&A] [answered]
How is the connectivity constraint implemented in torch_geometric/explain/algorithm
/pg_explainer.py  ?

**Comment by wsad1** [comment, marked as answer]
The [loss](https://github.com/pyg-team/pytorch_geometric/blob/4a7658f22cc16dfc4fc776195fb91d4257507c34/torch_geometric/explain/algorithm/pg_explainer.py#L242) function does not incorporate the connectivity constraint. This is because although the paper mentions connectivity contraints, the implementation in pyg is based on https://github.com/LarsHoldijk/RE-ParameterizedExplainerForGraphNeuralNetworks, which replicated the results in the paper.

**Reply by zuluokonkwo** [reply]
Thanks.
